python/ray/dashboard/agent.py

```diff
@@ -108,11 +108,15 @@ def _init_non_minimal(self):
                 ),
             )  # noqa
         )
-        grpc_ip = "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0"
+        grpc_ip = self.ip
```

Comment:
No need to introduce another variable, you can just directly use self.ip everywhere.